### ISSUE TITLE
android command line

### ISSUE THREAD

**Issue by zuck43** [lzhiyong/android-sdk-tools#41]
hey i need android command line can i use the Linux version of command line with this sdk our not here the link https://dl.google.com/android/repository/commandlinetools-linux-11076708_latest.zip

**Comment by lzhiyong** [comment]
yes, you can use the `cmdline-tools/bin/sdkmanager` to install the build-tools, platform-tools etc.

but you still need to replace them, since these binaries are for x84_64 architecture not aarch64
